model/src/pyrenew/latent/hospitaladmissions.py

```diff
@@ -14,30 +15,67 @@
     ["IHR", "predicted"],
     defaults=[None, None],
 )
-"""Output from HospitalAdmissions.sample()"""
+HospAdmissionsSample.__doc__ = """
+A container for holding the output from HospAdmissionsSample.sample.
+
+Attributes
+----------
+IHR : float or None
+    The infected hospitalization rate. Defaults to None.
+predicted : ArrayLike or None
+    The predicted number of hospital admissions. Defaults to None.
+
+Notes
+-----
+TODO: Add Notes.
+"""
 
 InfectHospRateSample = namedtuple(
     "InfectHospRateSample",
     ["IHR"],
     defaults=[None],
 )
+InfectHospRateSample.__doc__ = """
+A container for holding the output from InfectHospRateSample.sample.
+
+Attributes
+----------
+IHR : ArrayLike or None
+    The infected hospitalization rate. Defaults to None.
+
+Notes
+-----
+TODO: Add Notes.
+"""
 
 
 class InfectHospRate(RandomVariable):
-    """Infection to Hospitalization Rate"""
+    """
+    Infection to Hospitalization Rate
+
+    Methods
+    -------
+    validate(distr)
+        Validates distribution is Numpyro distribution
+    sample(**kwargs)
+        Produces a sample of the IHR
+    """
 
     def __init__(
         self,
-        dist: dist.Distribution,
-        varname: str = "IHR",
+        dist: Optional[dist.Distribution] = dist.LogNormal(
+            jnp.log(0.05), 0.05
+        ),
+        varname: Optional[str] = "IHR",
     ) -> None:
-        """Default constructor
+        """
+        Default constructor
```

Comment:
Just a question, I like the line breaks for starting docs, but I always see docstring starting right away (no like break after `"""`). What's the best practice here? cc @dylanhmorris and @natemcintosh

Reply:
If there is a best practice, I'm not aware of it. I always like to have a line break at the start of a docstring, if only to make reading it slightly easier. And the line break is gone when docs are generated (at least when I look at it using the VSCode hover to get docs)

Reply:
I like to start with a line break, and you do see it (although not universally) in public code. Here's an example from the source code for numpydoc itself 

https://github.com/numpy/numpydoc/blob/defebb1d5ffcef5f53c05a15091dbad2118ecd98/numpydoc/validate.py#L157-L174

Reply:
Thanks for confirming that!